server/athenian/api/internal/features/entries.py

```diff
@@ -494,6 +494,7 @@ async def batch_calc_pull_request_metrics_line_github(
                 quantiles,
                 self._quantile_stride,
                 exclude_inactive=exclude_inactive,
+                **(request.metric_params or {}),
```

Comment:
You see, having an empty default dict at object construction will simplify these places, we will be able to do `**request.metric_params` without thinking.